test/src/tests-identity.ts

```diff
@@ -34,17 +39,47 @@ declare global {
     }
 }
 
-let mockServer;
-const mParticle = window.mParticle;
+const mParticle = window.mParticle as MParticleWebSDK;
+
+const BadUserIdentitiesAsString = ({
```

Comment:
I agree - all caps for constants for best practices.

I think i just meant `BadUserIdentitiesString` `BadUserIdentitiesArray` which is what it was before.  But I don't really care, it can be kept as is.